linkerd2-policy-controller-edge.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: linkerd2-policy-controller-edge
```

Comment:
the naming of the package and file is done this way because this is coming from edge branch which is not stable release from LinkerD. It's the latest release from main and can contain bugs. Maintainers of LinkerD try their best to avoid this but still it's not stable. I am appending `-edge` is suffix for the same reason.